redis/src/lib.rs

```diff
@@ -369,6 +369,10 @@ pub use crate::connection::{
 pub use crate::parser::{parse_redis_value, Parser};
 pub use crate::pipeline::Pipeline;
 
+#[cfg(feature = "json")]
+#[cfg_attr(docsrs, doc(cfg(feature = "json")))]
+pub use crate::commands::json::Commands as JSONCommands;
```

Comment:
Looks good, but can we do the aliasing within the `commands` module itself, then you can just `use` the alias here. Also don't forget we'll need `AsyncCommands` aliased as well.

Reply:
> can we do the aliasing within the commands module itself?

yes, I'll get on that
> don't forget we'll need AsyncCommands aliased as well

ah whoops, will fix